packages/tse-backend/src/modules/exchange-operation/model/exchange-operation.model.ts

```diff
@@ -21,7 +21,7 @@ export class CreateLimitOrderCommand extends CreateMarketOrderCommand {
 export type OrderCommand = CreateMarketOrderCommand | CreateLimitOrderCommand;
 
 export class ExchangeOperationCommand {
-  orderEntityId: number;
+  orderEntityId?: number | undefined;
```

Comment:
Why can:  `orderEntityId?` be undefined?
In which scenario?

Reply:
`packages/tse-backend/src/modules/exchange-trade/exchange-trade.service.ts`
The `cancelOrder` method uses `orderId` to cancel the order, and to update the order status we don't need orderEntityId, because we get the entity using `orderId`.

Reply:
Then maybe it's better to create a separate DTO for canceling?

Reply:
Fixed: 0b747d9f